modules/core/shared/src/main/scala/gem/math/Offset.scala

```diff
@@ -79,9 +79,9 @@ object Offset {
     implicit val ShowP: Show[P] =
       Show.fromToString
 
-    /** P components are equal if their angles are equal. */
-    implicit val EqualP: Eq[P] =
-      Eq.by(_.toAngle)
+    /** P components are by signed angle. */
+    implicit val OrderP: Order[P] =
+      Order.by(_.toAngle.toSignedMicroarcseconds)
```

Comment:
Any reason why these aren't `Angle.SignedAngleOrder.contramap(_.toAngle)`?

Reply:
Didn't occur to me, but now that you mention it I like that better!

Reply:
I just didn't want to define the same ordering twice.